www/source/partials/docs/_using-hab-encryption.html.md.erb

```diff
@@ -23,20 +23,20 @@ Supervisors running in a ring can be configured to encrypt all traffic between t
 3. Start the Supervisor with the `-r` or `--ring` parameter, specifying the name of the ring key to use.
 
     ```shell
-    $ hab start --ring yourringname yourorigin/yourapp
+    $ hab sup run --ring yourringname yourorigin/yourapp
     ```
 
     or, if using a standalone Supervisor package:
 
     ```shell
-    $ hab-sup start --ring yourringname yourorigin/yourapp
+    $ hab-sup run --ring yourringname yourorigin/yourapp
```

Comment:
I'm actually wondering if it's even worth including direct use of the underlying `hab-sup` binary in our documentation. I feel like the fact that it exists as a separate binary is more of an implementation detail than something that end-users should be concerned about.

Any thoughts on that, @reset?

Reply:
I agree, we should use `hab sup` over `hab-sup` (and the like) everywhere.